network-api/networkapi/wagtailpages/tests/buyersguide/test_homepage.py

```diff
@@ -259,3 +259,41 @@ def test_get_editorial_content_index_no_such_page(self):
         result = self.bg.get_editorial_content_index()
 
         self.assertEqual(result, None)
+
+    def test_bg_home_page_with_cta(self):
+        url = self.bg.url
```

Comment:
**Question** Why is this assignment needed?

Reply:
Per our discussion, updated to use self.bg.url when it makes sense to.